model/metrics/v8js-metrics.yaml

```diff
@@ -14,10 +14,15 @@ groups:
   - id: metric.veightjs.memory.total
     type: metric
     metric_name: v8js.memory.total
-    brief: "Total memory size."
+    brief: "Total memory size pre-allocated."
     instrument: updowncounter
     unit: "By"
     stability: experimental
+    attributes:
+      - ref: v8js.memory.type
+        requirement_level: recommended
+      - ref: v8js.heap.space.name
+        requirement_level: recommended
```

Comment:
Given the comment:

> Available only when memory type is heap

Should the requirement level `v8js.heap.space.name` here be conditionally required? So only there if `memory.type = heap`?

Reply:
Updated